src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card"
+         data-qa="card"
+    >
+
+      <div class="picture-box">
+        <div class="picture-bem1">
+          <img
+            class="picture-bem1__elem"
+            src="images/imac.jpeg"
+            alt="Product's picture"
+          >
+        </div>
+      </div>
+
+      <div class="card-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="code-text">Product code: 195434</div>
+
+      <div class="star">
+        <div class="bem-stars">
+          <div class="bem-stars__star bem-stars__star-row--active1"></div>
+          <div class="bem-stars__star bem-stars__star-row--active2"></div>
+          <div class="bem-stars__star bem-stars__star-row--active3"></div>
+          <div class="bem-stars__star bem-stars__star-row--active4"></div>
+          <div class="bem-stars__star bem-stars__star-row--passive5"></div>
+        </div>
```

Comment:
please, fix it. Mentor mentioned on it 3 times, and even gived you solution
![image](https://user-images.githubusercontent.com/59209337/212004162-4a59beef-e22c-470c-bc3d-eb681b1542fd.png)

Reply:
I need numbers to make the pick up element more easy and clear, could I name like stars__star1, stars__star2 ?